addons/escoria-core/game/core-scripts/esc/esc_event_manager.gd

```diff
@@ -281,9 +281,6 @@ func _on_event_finished(finished_statement: ESCStatement, return_code: int, chan
 	if event_flags & ESCEvent.FLAG_NO_SAVE:
 		escoria.save_manager.save_enabled = true
 
-	if return_code == ESCExecution.RC_CANCEL:
```

Comment:
@dploeger can you maybe tell us why this block was here originally? It seems to just swallow a cancelled execution for some reason.